lib/capistrano-nvm/version.rb

```diff
@@ -1,3 +1,3 @@
 module CapistranoNvm
-  VERSION = '0.0.7'
+  VERSION = '0.0.9'.freeze
```

Comment:
We can move to 0.1.0 I think.

Reply:
Heh. I actually had to check the Semver spec before I realized that you're absolutely right about that 😆